Add domain expiry check script

vyshnavlalqburst opened the pull request:
Added bash script to check the domain expiry and send alerts. Also updated the README.md file under scripts.

qburst-praven commented on scripts/domain-expiry-check/domain_expiry_checker.sh:
```diff
@@ -0,0 +1,90 @@
+
+# Specify the username
+username=""
+
```
while I understand the logic, I think both this and the home_directory is useless, and should be removed. You can just add the correct location in domain_list variable.

vyshnavlalqburst commented:
Updated the script as per the comments. Removed `username` and `home_directory` variables and updated `domain_list` variable to get the correct location.
